ts/parts/Chart.ts

```diff
@@ -2885,6 +2886,20 @@ extend(Chart.prototype, /** @lends Highcharts.Chart.prototype */ {
 
         // Don't run again
         this.onload = null as any;
+    },
+
+    /**
+    * Check whether the chart has vertical panning ('y' or 'xy' type).
+    *
+    * @private
+    * @function Highcharts.Chart#hasVerticalPanning
+    * @return {boolean}
+    *
+    */
+    hasVerticalPanning: function (
```

Comment:
This function is only used in `Axis.ts`. To avoid polluting the Chart class, move it over to Axis.ts and make it a local function there.